pf_driver/src/pf/pf_interface.cpp

```diff
@@ -272,52 +272,47 @@ void PFInterface::reconfig_callback_r2000(pf_driver::PFDriverR2000Config& config
 
 void PFInterface::reconfig_callback_r2300(pf_driver::PFDriverR2300Config& config, uint32_t level)
 {
-  if (product_ != "R2300")
-    return;
-  if (state_ != PFState::RUNNING)
-    return;
-  // Do we want to change the address and port at run-time?
-  // if(level == 16){
-  //   set_parameter({ KV("address", config.address) });
-  // } else if(level == 17)
-  // {
-  //   set_parameter({ KV("port", config.port) });
-  // }
-  if (level == 18)
-  {
-    config_.packet_type = config.packet_type;
-  }
-  else if (level == 19)
-  {
-    // currently always none for R2300
-    // config_.packet_crc = config.packet_crc;
-  }
-  else if (level == 20)
-  {
-    config_.watchdog = (config.watchdog == "on") ? true : false;
-  }
-  else if (level == 21)
-  {
-    config_.watchdogtimeout = config.watchdogtimeout;
-  }
-  else if (level == 22)
-  {
-    config_.start_angle = config.start_angle;
-  }
-  else if (level == 23)
-  {
-    config_.max_num_points_scan = config.max_num_points_scan;
-  }
-  else if (level == 24)
-  {
-    config_.skip_scans = config.skip_scans;
-  }
-  else
-  {
-    protocol_interface_->handle_reconfig(config, level);
-  }
-  pipeline_->set_scanoutput_config(config_);
-  config_ = protocol_interface_->get_scanoutput_config(info_.handle);
-  params_ = protocol_interface_->get_scan_parameters(config_.start_angle);
-  pipeline_->set_scan_params(params_);
+
+    if(product_ != "R2300")
+        return;
+    if(state_ != PFState::RUNNING)
+        return;
+    // Do we want to change the address and port at run-time?
+    // if(level == 16){
+    //   set_parameter({ KV("address", config.address) });
+    // } else if(level == 17)
+    // {
+    //   set_parameter({ KV("port", config.port) });
+    // }
+    if(level == 18)
+    {
+        config_.packet_type = config.packet_type;
+    } else if(level == 19)
+    {
+        // currently always none for R2300
+        // config_.packet_crc = config.packet_crc;
+    } else if(level == 20)
+    {
+        config_.watchdog = (config.watchdog == "on") ? true : false;
+    } else if(level == 21)
+    {
+        config_.watchdogtimeout = config.watchdogtimeout;
+    } else if(level == 22)
+    {
+        config_.start_angle = config.start_angle;
+    } else if(level == 23)
+    {
+        config_.max_num_points_scan = config.max_num_points_scan;
+    } else if(level == 24)
+    {
+        config_.skip_scans = config.skip_scans;
+    } else
+    {
+        protocol_interface_->handle_reconfig(config, level);
+    }
+    pipeline_->set_scanoutput_config(config_);
+    protocol_interface_->set_scanoutput_config(info_.handle,config_);
+    config_ = protocol_interface_->get_scanoutput_config(info_.handle);
+    params_ = protocol_interface_->get_scan_parameters(config_.start_angle);
+    pipeline_->set_scan_params(params_);
```

Comment:
are these changes only because of indentation? Please take care not to introduce white-spaces.